extract/tests/otto/otto_test.py

```diff
@@ -0,0 +1,58 @@
+from requests_mock import Adapter
+from tests.test_utils import read_test_html
```

Comment:
better use relative import: `../test_utils`. You need to create `__init__.py` files in `tests` and `tests/otto`